filter/series_by_tag_pattern_index_test.go

```diff
@@ -64,14 +66,14 @@ func TestSeriesByTagPatternIndex(t *testing.T) {
 			"dc!~=ru":          {{"dc", NotMatchOperator, "ru"}},
 			"invalid operator": {{"dc", TagSpecOperator("invalid operator"), "ru"}},
 
-			"name=cpu1;dc=ru1": {{"name", MatchOperator, "cpu1"}, {"dc", EqualOperator, "ru1"}},
-			"name=cpu1;dc=ru2": {{"name", MatchOperator, "cpu1"}, {"dc", EqualOperator, "ru2"}},
-			"name=cpu2;dc=ru1": {{"name", MatchOperator, "cpu2"}, {"dc", EqualOperator, "ru1"}},
-			"name=cpu2;dc=ru2": {{"name", MatchOperator, "cpu2"}, {"dc", EqualOperator, "ru2"}},
-			"name=disk;dc=ru1": {{"name", MatchOperator, "disk"}, {"dc", EqualOperator, "ru1"}},
-			"name=disk;dc=ru2": {{"name", MatchOperator, "disk"}, {"dc", EqualOperator, "ru2"}},
-			"name=cpu1;dc=us":  {{"name", MatchOperator, "cpu1"}, {"dc", EqualOperator, "us"}},
-			"name=cpu2;dc=us":  {{"name", MatchOperator, "cpu2"}, {"dc", EqualOperator, "us"}},
+			"name=cpu1;dc=ru1": {{"name", EqualOperator, "cpu1"}, {"dc", EqualOperator, "ru1"}},
```

Comment:
Why are there changes in the test?

Reply:
The patterns were incorrect. Keys and values in tagSpecsByPattern should contain the same data in different types. Because in production code second will be generated from first.
 "name=cpu1" corresponds to EqualOperator, "name~=cpu1" corresponds to MatchOperator.